src/Lykke.Job.OpsGenie/Startup.cs

```diff
@@ -0,0 +1,191 @@
+
+            Configuration = builder.Build();
+            Environment = env;
+        }
+
+        public IServiceProvider ConfigureServices(IServiceCollection services)
+        {
+            try
+            {
+                services.AddMvc()
+                    .AddJsonOptions(options =>
+                    {
+                        options.SerializerSettings.Converters.Add(new StringEnumConverter());
+                        options.SerializerSettings.ContractResolver = new Newtonsoft.Json.Serialization.DefaultContractResolver();
+                    });
+
+                services.AddSwaggerGen(options =>
+                {
+                    options.DefaultLykkeConfiguration(ApiVersion, ApiName);
+                });
+
+                var settingsManager = Configuration.LoadSettings<AppSettings>(options =>
+                {
+                    options.SetConnString(x => x.SlackNotifications.AzureQueue.ConnectionString);
+                    options.SetQueueName(x => x.SlackNotifications.AzureQueue.QueueName);
+                    options.SenderName = $"{AppEnvironment.Name} {AppEnvironment.Version}";
+                });
+
+                var appSettings = settingsManager.CurrentValue;
+
+                if (appSettings.MonitoringServiceClient != null)
+                    _monitoringServiceUrl = appSettings.MonitoringServiceClient.MonitoringServiceUrl;
+
+                services.AddLykkeLogging(
+                    settingsManager.ConnectionString(s => s.OpsGenieJob.Db.LogsConnString),
+                    "OpsGenieLog",
+                    appSettings.SlackNotifications.AzureQueue.ConnectionString,
+                    appSettings.SlackNotifications.AzureQueue.QueueName);
+
+                var builder = new ContainerBuilder();
+                builder.Populate(services);
+
+                builder.RegisterModule(new JobModule(appSettings.OpsGenieJob, settingsManager.Nested(x => x.OpsGenieJob)));
+
+                ApplicationContainer = builder.Build();
+
+                var logFactory = ApplicationContainer.Resolve<ILogFactory>();
+                _log = logFactory.CreateLog(this);
+                _healthNotifier = ApplicationContainer.Resolve<IHealthNotifier>();
+
+                return new AutofacServiceProvider(ApplicationContainer);
+            }
+            catch (Exception ex)
+            {
+                if (_log == null)
+                    Console.WriteLine(ex);
+                else
+                    _log.Critical(ex);
+                throw;
+            }
+        }
+
+        public void Configure(IApplicationBuilder app, IHostingEnvironment env, IApplicationLifetime appLifetime)
+        {
+            try
+            {
+                if (env.IsDevelopment())
+                    app.UseDeveloperExceptionPage();
+
+                app.UseLykkeForwardedHeaders();
+                app.UseLykkeMiddleware(ex => new ErrorResponse {ErrorMessage = "Technical problem"});
```

Comment:
Build the message from  the ex.ToAsyncString()

Reply:
Do we need to return full stacktrace here? Now its  omitted